### PR TITLE
feat: Added jsonplaceholder example

### PR DESCRIPTION
**Pull request by jfollmann**
Hi @camilamaia,

I added another example using the jsonplaceholder API (https://jsonplaceholder.typicode.com). Issue #5 .

Congratulations on the excellent work. :heart_eyes:

### CODE REVIEW

**Comment by jfollmann** [comment]
note: two requests are showing errors, because the API is in trouble... hehe

File: jsonplaceholder-api/resources/users.yaml [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+      assert: ${{ response.status_code == 201 }}
+    
+    - name: response_time_is_under_a_second
+      assert: ${{ response.elapsed.total_seconds() < 1 }}
+
+- name: update_with_put
+  method: put
+  path: ${newRecordId}
+  body:
+    # id: ${newRecordId}
```
Comment (camilamaia) [inline jsonplaceholder-api/resources/users.yaml:63]: Ops, I thinks you forgot a comment here.

**Review by camilamaia — COMMENTED** [review]
Wow, this is a super complete example! Thanks for that 😍 

We were missing a complete example with lots of different http methods and multiple spec files. Thanks for the contribution!

I just left a minor comment, but overall it is awesome!

**Comment by camilamaia** [comment]
> note: two requests are showing errors, because the API is in trouble... hehe

Wow, super interesting to see how the API is breaking for PUT methods!

**Review by camilamaia — APPROVED** [review]
⭐

**Comment by camilamaia** [comment]
Hey @jfollmann, I am gonna send you an invite to join the ScanAPI org on GitHub. We invite everyone that has contributed with a merged PR in any of our repos. Here you can check our [Community Continuity Guidelines ](https://github.com/scanapi/contributors/blob/master/Contributing.md) so you can understand better how it works.

And once again, thank you very much!! 🙇‍♀️

**Comment by jfollmann** [comment]
Thanks @camilamaia. I hope to be able to contribute more. 😊

**Pull request merged**